types/office-js-preview/index.d.ts

```diff
@@ -21185,6 +21264,16 @@ declare namespace Excel {
          * [Api set: ExcelApi 1.1]
          */
         readonly rowIndex: number;
+        /**
+         *
+         * Represents if ALL the cells would be saved as an array formula
```

Comment:
If this all gets wrapped together, then I recommend a period at the end of this sentence.

```suggestion
         * Represents if ALL the cells would be saved as an array formula.
```

Reply:
ditto for later occurrences.